src/rpc/coinjoin.cpp

```diff
@@ -440,6 +440,16 @@ static RPCHelpMan getcoinjoininfo()
                                                 {RPCResult::Type::STR, "address", ""},
                                             }
                                         },
+                                        {RPCResult::Type::ARR, "platform_p2p", /*optional=*/true, "Addresses used for Platform P2P",
```

Comment:
nit: if you will keep it here, consider mentioning evo nodes here; 
maybe something like
```
                                        {RPCResult::Type::ARR, "platform_p2p", /*optional=*/true, "Addresses used for Platform P2P (EvoNodes only)",
```

Reply:
Resolved in latest push